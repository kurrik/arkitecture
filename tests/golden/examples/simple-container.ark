# Simple container with children
parent {
  label: "Parent Container"
  direction: "vertical"
  
  child1 {
    label: "Child One"
  }
  
  child2 {
    label: "Child Two"
  }
}