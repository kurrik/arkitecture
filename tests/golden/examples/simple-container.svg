<svg xmlns="http://www.w3.org/2000/svg" width="88" height="64">
  <defs>
    <marker id="arrowhead" markerWidth="10" markerHeight="7" 
            refX="9" refY="3.5" orient="auto" markerUnits="strokeWidth">
      <polygon points="0 0, 10 3.5, 0 7" fill="black" />
    </marker>
  </defs>

  <!-- Node rectangles and labels -->
  <rect x="0" y="0" width="88" height="64" fill="white" stroke="black" stroke-width="1" />
  <text x="44" y="32" text-anchor="middle" dominant-baseline="middle" font-family="Helvetica" font-size="16">Parent Container</text>
  <rect x="0" y="0" width="88" height="32" fill="white" stroke="black" stroke-width="1" />
  <text x="44" y="16" text-anchor="middle" dominant-baseline="middle" font-family="Helvetica" font-size="16">Child One</text>
  <rect x="0" y="32" width="88" height="32" fill="white" stroke="black" stroke-width="1" />
  <text x="44" y="48" text-anchor="middle" dominant-baseline="middle" font-family="Helvetica" font-size="16">Child Two</text>
</svg>