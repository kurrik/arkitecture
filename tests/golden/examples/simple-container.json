{
  "fontSize": 16,
  "fontFamily": "Helvetica"
}